Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -142,37 +148,41 @@ public static IEnumerable<string> FindPluginAssemblies(string pluginFolder, bool
                 {
                     continue;
                 }
+                
+                string extensionFolder = Path.GetDirectoryName(assemblyPath);
+                string assemblyMetadataPath = Directory.GetFiles(

```

Comment:
The requirement on [the ticket](https://tracker.ed-fi.org/browse/ODS-4664) is for the _assemblyMetadata.json_ file to be an embedded resource in the plugin assembly.

You'll need to use the `Assembly` class' `GetManifestResourceNames` and `GetManifestResourceStream` methods to get the resource from the assembly. Then you can use a `StreamReader` and its `ReadToEnd` method (see documentation [here](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader.readtoend?view=net-5.0)) to get the JSON for deserialization.

This will also require changes to the projects in the _Ed-Fi-Extensions repository_, and also the sample Profiles project (?).